crates/pallet-subspace/src/lib.rs

```diff
@@ -622,6 +641,41 @@ pub mod pallet {
 
             Ok(())
         }
+
+        /// Update proof of time slot iterations
+        #[pallet::call_index(6)]
+        #[pallet::weight(< T as Config >::WeightInfo::set_pot_slot_iterations())]
+        pub fn set_pot_slot_iterations(
+            origin: OriginFor<T>,
+            slot_iterations: NonZeroU32,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+
+            if slot_iterations.get() % u32::from(PotCheckpoints::NUM_CHECKPOINTS.get() * 2) != 0 {
+                return Err(Error::<T>::NotMultipleOfCheckpoints.into());
+            }
+
+            if PotSlotIterations::<T>::get().expect("Always initialized during genesis; qed")
```

Comment:
Do we extend ["no panic in runtime code" rule](https://docs.substrate.io/build/troubleshoot-your-code/) to `expect` as well? @vedhavyas

Reply:
We generally do not expect panics in runtime just like any other code. This particular line is guaranteed to not panic because as it says we initialize it during genesis, specifically see line 352 above that is called when chain starts to do any necessary initial setup.